### PR TITLE
tweaked the travis yml config

### PR DESCRIPTION
**Pull request by Holajuwon**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,15 +2,16 @@ language: node_js
...
-
 install:
   - npm ci
-
 script:
   - npm test
-
 deploy:
-  skip-cleanup: true
-  github-token: $GITHUB-TOKEN
```
Comment (Tboy-AK) [inline .travis.yml:14]: Why was the github token variable removed, which seems to be of utmost essence?

File: .travis.yml [hunk 2]
```diff
@@ -2,15 +2,16 @@ language: node_js
...
 cache: npm
-
 install:
   - npm ci
-
 script:
   - npm test
-
 deploy:
-  skip-cleanup: true
```
Comment (Tboy-AK) [inline .travis.yml:13]: Why was the skipping of cleanup discarded?

File: .travis.yml [hunk 3]
```diff
@@ -2,15 +2,16 @@ language: node_js
...
-  github-token: $GITHUB-TOKEN
+  cleanup: true
+  provider: heroku
+  api_key:
+    secure: XrlNI0t4pd8TbwsnTprKDunnx+L7BnHbUpMjjQrdsJ+MwqfnQlJzyKZYbpbSSx/WO3twY2McFDHe93DchxzYhU0kC2KJe/N7zHera4pyteaMqpQQQmPekqFFZmzceXK3nLAGNVOGPCzA9JWH/31580D0NKo9Q4oG2UVMq00zZP82CgJ3zIM3V1eUaSXA0yjSMdWbpUG3/2fJHqjeyB0HAoXN/3EaAp72b1tVmoF0M5nJiQ0MAESln7ek8UiiNGXHNZiRBMskUanMbSF69WCiSsp3Yae9pnXAyDFRtB1KlnMMO3+IMmZ1PQxARV945z84I6FWtdr+HLXAnyj2j1rKyZleKXv7jmQHjT1/Mj/0JWMHYg/Mi0880mqY125uC1c+81XwOtPq1ub/IvoEYPDke65TV7GtPP/DGkvPmIVAkhz0NB6yHl5X4mdBvXs+TjD2GkjG81sPnKH4Jcy80y5XJAwRyyKrcnE9AjPKSccbKhC2Oo+Yltk9M2SFNMObV1kJUW65G5Ge7LgoyvWBsKIoUocQ2VTdLIqxdKqFBGQ6U3fcBP20VKTVPPVkbBVBd7k8nRxZ25PEeTSvC0hQw6WsqdIjg87VnESxlkx/CTC+sjgC+RKClQeU8tZx9XEIFy445ccU/otsCCi0N3jYbCpKy7teWNymBJ2OJDij0ux86kU=
+  app: howmies-api-develop
   on:
+    repo: Matthewnte/howmies-api
```
Comment (Tboy-AK) [inline .travis.yml]: You forgot to change the repo owner

**Review by Tboy-AK — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (Holajuwon) [reply .travis.yml:14]: It's not actually required for deployment from what I did earlier,
Actually, there's an automated Travis command to setup Heroku deployment,
```bash
$ travis setup heroku
```
It overrides the previous setting there.
image
Also from the image above, it shows that the key is unknown.

File: .travis.yml (hunk 2 above)
Comment (Holajuwon) [reply .travis.yml:13]: skip-cleanup as been deprecated by Travis its just cleanup now, I think I added it but now I see, I should have set it to false based on the skip-cleanup.

File: .travis.yml (hunk 3 above)
Comment (Holajuwon) [reply .travis.yml]: That's a mistake, Travis autogenerated based on the repo name in the git tree, I will fix it.
Has the Travis been approved for the organization?

File: .travis.yml (hunk 1 above)
Comment (Tboy-AK) [reply .travis.yml:14]: The github token was setup as a Travis env. We will have to get a new one later, but for now, we will keep that on hold and continue with your implementation. After we have fully and successfully incorporated your implementation, we will then test the deployment with the new GitHub token. Well done @Holajuwon !

File: .travis.yml (hunk 3 above)
Comment (Tboy-AK) [reply .travis.yml]: Still working on it. Once it is done, you will be informed.

File: .travis.yml (hunk 1 above)
Comment (Tboy-AK) [reply .travis.yml:14]: The github token was setup as a Travis env. We will have to get a new one later, but for now, we will keep that on hold and continue with your implementation. After we have fully and successfully incorporated your implementation, we will then test the deployment with the new GitHub token. Well done @Holajuwon !

**Pull request closed without merging**